src/AppOriginal.js

```diff
@@ -0,0 +1,171 @@
+import React from "react";
```

Comment:
I am a bit confused. What is the difference between App.js and AppOriginal.js? I only just noticed those are 2 different files, which makes my review a bit unreliable. Please only keep 1 copy of a component in your repositories, as this would confuse anyone reviewing or reading your code